### PR TITLE
Adding ioBanker gateway codes

### PR DESCRIPTION
**Pull request by ioBanker**

### CODE REVIEW

File: app/actions/GatewayActions.js [hunk 1]
```diff
@@ -18,6 +18,57 @@ const onGatewayTimeout = (dispatch, gateway) => {
 };
 
 class GatewayActions {
+    fetchCoins({
```
Comment (sschiessl-bcp) [inline app/actions/GatewayActions.js]: What's different to the other method?

Comment (ioBanker) [reply app/actions/GatewayActions.js]: Which modal? Which other method?

Comment (ioBanker) [reply app/actions/GatewayActions.js]: Please consider that there is no description for the codes, I cannot co-relate everything, I am trying to do it the standard way.

Comment (sschiessl-bcp) [reply app/actions/GatewayActions.js]: In the same file there are two other "fetchCoins" methods. Difference?

**Comment by sschiessl-bcp** [comment]
The modal is the "not legacy" deposit

image

File: app/actions/GatewayActions.js (hunk 1 above)
Comment (ioBanker) [reply app/actions/GatewayActions.js]: This file pull needs to be cancelled, I shouldn't have added this

**Comment by ioBanker** [comment]
> The modal is the "not legacy" deposit
> 
> image

I believe you are talking about this https://github.com/bitshares/bitshares-ui/pull/3165/commits/41df7f9c6c8352dd551104b7c92474a0dcea42e7

**Comment by sschiessl-bcp** [comment]
If you think you have misplaced some code I would suggest to do a fresh checkout of develop branch locally, then add all code you need to have and then do a PR. Or is this now complete from your end?

**Comment by ioBanker** [comment]
> If you think you have misplaced some code I would suggest to do a fresh checkout of develop branch locally, then add all code you need to have and then do a PR. Or is this now complete from your end?

This is complete from my end, I will add common changes to the whole code soon later not related to my gateway codes only.

**Comment by sschiessl-bcp** [comment]
Stale, if not please reopen

**Pull request closed without merging**